example/ssd/demo.py

```diff
@@ -109,6 +109,8 @@ def parse_args():
                         help='string of comma separated names, or text filename')
     parser.add_argument('--camera', action='store_true',
                         help="use camera for image capturing")
+    parser.add_argument('--frame-resize', type=str, default=None
```

Comment:
should this default to 1.0?

Reply:
None is fine

Reply:
We avoid a call to resizing if it's none